notebooks/test_notebooks.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+from io import open
+
+from nbformat import read, write
+from nbconvert.preprocessors import ExecutePreprocessor, CellExecutionError
+# Other dependencies: jupyter-client, ipykernel
+
+import logging
+import traceback
+
+
+'''
+Checks notebooks by trying to convert each one into a notebook
+that includes its execution results.
+
+This script is similar to this command:
+jupyter nbconvert --to notebook --execute --ExecutePreprocessor.timeout=60
```

Comment:
What's then the idea of this script ?

If we can execute :

```
jupyter nbconvert --to notebook --execute --ExecutePreprocessor.timeout=60 --output executed_notebook.ipynotebook demo.ipynb
```

We'll know that if it fails, then for example CircleCI should fail

Reply:
```
jupyter nbconvert --to notebook --execute --ExecutePreprocessor.timeout=60 --output executed_notebook.ipynotebook demo.ipynb
```

Why did you choose to limit timeout to 60 seconds ?